#! /bin/sh /usr/share/dpatch/dpatch-run
## 902_define_YAML_in_yaml_stringio.rb.dpatch by akira yamada <akira@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: YAML module is needed.  (It is a workaround for generating RDoc.)

@DPATCH@
Index: b/lib/yaml/stringio.rb
===================================================================
--- a/lib/yaml/stringio.rb	2010-07-20 21:45:46.000000000 +0900
+++ b/lib/yaml/stringio.rb	2010-07-20 21:46:14.000000000 +0900
@@ -49,6 +49,7 @@
         end
     end
 
+    module YAML
 	#
 	# Class method for creating streams
 	#
@@ -78,6 +79,6 @@
         end
         io
 	end
-
+    end
 end
 
